src/content/ModalLayer.tsx

```diff
@@ -45,6 +48,12 @@ interface ModalAction {
   additionalData?: unknown
 }
 
+interface SyncScheduleModalData {
+  data: [""]
```

Comment:
oops yea did mean ``String[]`` Named it ``data`` because I was planning on using it to pass data for other sync modals but didnt end up using them, made it more descriptive now